docs/auto-instrumentation-install.md

```diff
@@ -0,0 +1,278 @@
+```
+
+### 2. Deploy the Helm Chart with the Operator enabled
+
+You can pass `--set operator.enabled=true` when deploying the chart to enable the operator. You can also use commands
+like in the following example.
+
+```
+VALUES_FILE=examples/enable-operator-and-auto-instrumentation/enable-operator-and-auto-instrumentation-values.yaml
+helm install splunk-otel-collector \
+-f $VALUES_FILE \
+-n monitoring \
+helm-charts/splunk-otel-collector
+```
+
+### 3. Deploy the opentelemetry.io/v1alpha1 Instrumentation
+
+This Instrumentation object is a spec to configure what instrumentation libraries to use for instrumentation. An
+Instrumentation object must be available to the target pod for auto-instrumentation to function. Here is an example
+of a Instrumentation yaml file and how to install it.
+
+<details open>
+<summary>Expand for sample splunk-instrumentation.yaml</summary>
+
+```yaml
+# splunk-instrumentation.yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: splunk-instrumentation
+spec:
+  exporter:
+    endpoint: http://$(SPLUNK_OTEL_AGENT):4317
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+  dotnet:
+    env:
+      - name: SPLUNK_OTEL_AGENT
+        valueFrom:
+          fieldRef:
+            apiVersion: v1
+            fieldPath: status.hostIP
+      - name: OTEL_EXPORTER_OTLP_ENDPOINT
+        value: http://$(SPLUNK_OTEL_AGENT):4317
+      - name: OTEL_TRACES_EXPORTER
+        value: otlp
+```
+
+</details>
+
+```
+# Install
+kubectl apply -f splunk-instrumentation.yaml
+# Check the current deployed values
+kubectl get otelint -o yaml
+```
+
+### 4. Verify all the OpenTelemetry resources (collector, operator, webhook, instrumentation) are deployed successfully
+
+<details open>
+<summary>Expand for sample output to verify against</summary>
+
+```
+kubectl  get pods -n monitoring
+# NAME                                                          READY
+# STATUS    RESTARTS   AGE
+# splunk-otel-collector-agent-9ccgn                             2/2     Running   0          3m
+# splunk-otel-collector-agent-ft4xc                             2/2     Running   0          3m
+# splunk-otel-collector-k8s-cluster-receiver-56f7c9cf5b-mgsbj   1/1     Running   0          3m
+# splunk-otel-collector-operator-6dffc898df-5jjkp               2/2     Running   0          3m
+
+kubectl get mutatingwebhookconfiguration.admissionregistration.k8s.io -n monitoring
+# NAME                                      WEBHOOKS   AGE
+# cert-manager-webhook                      1          8m
+# splunk-otel-collector-operator-mutation   3          2m
+
+kubectl get otelinst -n spring-petclinic
+# NAME                          AGE   ENDPOINT
+# splunk-instrumentation        3m   http://$(SPLUNK_OTEL_AGENT):4317
+```
+
+</details>
+
+### 5. Instrument application by setting an annotation
+
+An _inject instrumentation_ annotation can be added to the following.
```

Comment:
Is the annotation name `inject instrumentation`? Isn't it `instrumentation.opentelemetry.io/inject-java`?

Reply:
Other annotation names include:
instrumentation.opentelemetry.io/inject-python
instrumentation.opentelemetry.io/inject-nodejs
instrumentation.opentelemetry.io/inject-dotnet
more...

I'll add more specs and supported annotation names to this areas.

Reply:
Done